### PR TITLE
For Credential Application processing, handle JWT and Data Integrity Creds

### PR DESCRIPTION
**Pull request by decentralgabe**

### CODE REVIEW

File: credential/manifest/model.go [hunk 1]
```diff
@@ -322,6 +323,36 @@ func IsValidCredentialApplicationForManifest(cm CredentialManifest, applicationA
 	return nil
 }
 
+// turn a generic cred into a known shape without maintaining the proof/signature wrapper
+func credentialsFromInterface(genericCred interface{}) (*credential.VerifiableCredential, error) {
```
Comment (nitro-neal) [inline credential/manifest/model.go:327]: awesome, like if we want to refactor one day, this function can be used at the very first line in the service.

The service input gives us 'something' we instantly take that something and turn it into a *credential.VerifiableCredential. We can do this for all objects that can take different forms

Comment (nitro-neal) [reply credential/manifest/model.go:327]: so maybe make it capital CredentialsFromInterface because I have a feeling we will use this somewhere else

Comment (decentralgabe) [reply credential/manifest/model.go:327]: agreed...there is more refactoring to do here and moving a number of things from the service here

**Review by nitro-neal — APPROVED** [review]

File: credential/manifest/model.go (hunk 1 above)
Comment (decentralgabe) [reply credential/manifest/model.go:327]: moving to a new pkg in credentials called `util` to avoid import cycles

**Comment by decentralgabe** [comment]
added tests

**Pull request merged**